aptos-move/framework/supra-framework/sources/coin.move

```diff
@@ -402,7 +402,7 @@ module supra_framework::coin {
         fungible_asset::mint_internal(metadata, amount)
     }
 
-    /// A public wrapper around `coin_to_fungible_asset_internal` for use by external tests.
+    // A public wrapper around `coin_to_fungible_asset_internal` for use by external tests.
     #[test_only]
```

Comment:
@axiongsupra , The use of `///` is not the issue here. The warning occurs because the `#[test_only]` annotation is defined after the comment. It should be placed before the comment to resolve the warning.

Reply:
Thanks @nizam-supraoracles